README.md

```diff
@@ -3,30 +3,62 @@
 
 This repository includes the recipe for deploying a Tella Web instance (frontend and backend).
```

Comment:
Just for context, I'd link here to the user facing documentation of Tella Web: https://tella-app.org/tella-web

Highlighting that this README is just a guide on how to install the server, but that more info about how to manage it or to operate (with step-by-step guides) it can be found on that website (in case someone comes here looking fot the other thing